hack/make/release-deb

```diff
@@ -35,6 +35,51 @@ if [[ ! -f "$APTDIR/conf/distributions" ]]; then
 	done > "$APTDIR/conf/distributions"
 fi
 
+# create/update distributions file
+if [[ ! -f "$APTDIR/conf/apt-ftparchive.conf" ]]; then
```

Comment:
Just going to nitpick to see if @tianon corrects me:)

I think this only needs a single bracket